packages/summary-table-react/documentation/SummaryTableExample.tsx

```diff
@@ -16,15 +16,49 @@ const SOFT_HYPHEN = "\u00AD";
 
 export const summaryTableExampleCode: CodeExample = `
 <SummaryTable
-    columnDescriptions={["Produkt", "Pris"]}
-    items={[
-        { label: "Person", value: "", bold: true },
-        { label: "Barne${SOFT_HYPHEN}forsikring", value: "200 kr/mnd" },
-        { label: "Behandlings${SOFT_HYPHEN}forsikring", value: "200 kr/mnd" },
-        { label: "Skade", value: "", bold: true },
-        { label: "Innboforsikring", value: "100 kr/mnd" },
-        { label: "Reiseforsikring", value: "100 kr/mnd" },
-    ]}
-    footer={{ label: "Total sum", value: "600 kr/mnd" }}
+    caption="Informasjon om produkter og pris i handlekurven"
+    header={["Produkt", "Pris"]}
+    body={
+        <tbody>
+            <tr>
+                <th scope="row">Uføre${SOFT_HYPHEN}pensjon</th>
+                <td>340,00 kr/mnd</td>
+            </tr>
+
+            <tr className="summary-table-example__custom-row">
+                <th scope="row">Livs${SOFT_HYPHEN}forsikring</th>
+                <td>Tatt ut av tilbudet</td>
+            </tr>
+
+            <tr>
+                <th scope="row">Innbo${SOFT_HYPHEN}forsikring</th>
+                <td>122,00 kr/mnd</td>
+            </tr>
+
+            <tr>
+                <th scope="row">Reise${SOFT_HYPHEN}forsikring</th>
+                <td>249,00 kr/mnd</td>
+            </tr>
+
+            <tr className="summary-table-example__custom-row">
+                <th scope="row">Rabatt</th>
+                <td>- 17,50 kr/mnd</td>
+            </tr>
+        </tbody>
+    }
+    footer={
+        <tfoot>
+            <tr>
+                <th scope="row">Total sum</th>
+
+                <td>693,50 kr/mnd</td>
+            </tr>
```

Comment:
Samme innvending her som over. Ville foretrukket en `SummaryTableFooterRow` elns, og at `<tfoot>` defineres i `SummaryTable` i stedet for manuelt.